app/src/org/commcare/provider/InstanceProvider.java

```diff
@@ -542,10 +542,14 @@ private void linkToSessionFormRecord(Uri instanceUri) {
             // before moving on. We'll catch any errors here and just eat them
             // (since the task will also try the process and fail if it does.
             if (FormRecord.STATUS_COMPLETE.equals(current.getStatus())) {
+                net.sqlcipher.database.SQLiteDatabase db =
```

Comment:
I'd rename this reference to "userDb" or something so that someone searching the file doesn't confuse which "db" object is being manipulated here (since most of hte time it's not this one, but they share the same name)